getgist/local.py

```diff
@@ -16,8 +16,15 @@ def __init__(self, filename, assume_yes=False):
         return: (None)
         """
         self.cwd = os.getcwd()
-        self.file_path = os.path.expanduser(filename)
-        self.filename = os.path.basename(filename)
+
+        if filename:
+            self.file_path = os.path.expanduser(filename)
+            self.filename = os.path.basename(filename)
+
+        else:
+            self.file_path = None
+            self.filename = None
+        
```

Comment:
I don't get why a command that does not use `LocalTools` has impacted `LocalTools`. Can you clarify, please?